tests/hypertrace/agent/config/test_agent-config.yaml

```diff
@@ -0,0 +1,27 @@
+service_name: pythonagent_001
+reporting:
+  endpoint: http://localhost:9411/api/v2/spans
```

Comment:
we should use OTLP for all our testing going forward.

Reply:
By default we use OTLP, this file is used in the tests to verify that file configuration overrides default config. 
I think we should keep this as-is?

Reply:
ah, ok.  yes, we should keep it.